src/Plugins/RpcServer/ParameterConverter.cs

```diff
@@ -39,7 +42,12 @@ static ParameterConverter()
             { typeof(bool), token => Result.Ok_Or(token.AsBoolean, CreateInvalidParamError<bool>(token)) },
             { typeof(UInt256), ConvertUInt256 },
             { typeof(ContractNameOrHashOrId), ConvertContractNameOrHashOrId },
-            { typeof(BlockHashOrIndex), ConvertBlockHashOrIndex }
+            { typeof(BlockHashOrIndex), ConvertBlockHashOrIndex },
+            { typeof(Signer), ConvertSigner },
+            { typeof(ContractParameter), ConvertContractParameter },
+            { typeof(Signer[]), ConvertSignerArray },
+            { typeof(ContractParameter[]), ConvertContractParameterArray },
+            { typeof(Guid), ConvertGuid }
```

Comment:
Not resolved.

Reply:
It can not be added here, its signature is different from other handler methods, it need setting as parameter, thus has to be handled seperately as UInt160.